index.html

```diff
@@ -375,26 +341,142 @@
 
     <section>
       <h2>Core Data Model</h2>
-      <p>This section describes a data model for <a>entity
-      profiles</a> and <a>entity credentials</a>, the latter covering
+
+      <p>
+This section outlines core data model concepts used throughout this
+specification
+      </p>
+
+      <section>
+        <h3>Claims</h3>
+
+        <p>
+A <a>subject</a> is an <a>entity</a> about which <a>claims</a> may be made.
+A <a>claim</a> is statement made by an <a>entity</a> about a <a>subject</a>.
```

Comment:
"made by an entity" is not in the model for a claim (see the diagram) so we should omit it, i.e. "A claim is a statement about a subject." This has been one of the challenges between mapping the "claim" language to implementations. Where "claim" is used in implementations is as a specific property of the vocabulary. It is used in a statement about a credential:

The credential C "includes the claim" that subject S is related to object O via property P.

So that looks like this as a graph:

`[ C ] --claim--> [ S ] --P--> [ O ]`